tests/Sdk/Acceptance/Extension/Tasks/AnalyzerReportTaskCest.php

```diff
@@ -23,6 +23,8 @@ class AnalyzerReportTaskCest
     protected const PROJECT_DIR = 'upgrader_success_project';
 
     /**
+     * @skip
```

Comment:
why? Please, make it work or describe a reason for skip. the same for the others

Reply:
these tests are for upgrader. I have no idea why.